### PR TITLE
Adding dark theme to fabric website

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Adding a dark theme to fabric website (There is no perspective for soft, neutral, strong variants)

Includes semantic slot naming updates for various components: `Persona`, `ContextualMenu,` `SplitButton`, `SpinButton`, `Tooltip`,`Dropdown`, `ComboBox`, `CommandBar`,`Suggestions`, 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11446)

### CODE REVIEW

**Comment by aneeshack4** [comment]
LGTM! I think this can promoted from draft PR we've been looking forward to dark theme

**Comment by marygans** [comment]
@aneeshack4 thanks for pinging this. I will finish up polish work with the designer today!

**Comment by marygans** [comment]
@aneeshack4 thanks for pinging this. I will check everything for this, this week.

**Comment by marygans** [comment]
Just did a merge + push on this PR, I believe it's ready to go :)

File: packages/office-ui-fabric-react/src/components/Divider/examples/VerticalDivider.Basic.Example.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { VerticalDivider } from 'office-ui-fabric-react/lib/Divider';
 import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { getTheme } from '@uifabric/styling';
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/Divider/examples/VerticalDivider.Basic.Example.tsx]: Combine this with the oufr/lib/Styling import above

**Pull request merged**